spec/unit/provider_resolver_spec.rb

```diff
@@ -482,7 +482,6 @@ def stub_service_configs(*configs)
     python: Chef::Provider::Script,
     remote_directory: Chef::Provider::RemoteDirectory,
     route: Chef::Provider::Route,
-    rpm_package: Chef::Provider::Package::Rpm,
```

Comment:
(specifically, there is a line further down that says rpm_package: nil, so all OS's verify that rpm_package does *not* work, except the OS's on which it is supposed to work. This preserves behavior before the patch, no changes.)